docs/api/batch.md

```diff
@@ -1,5 +1,7 @@
 # Git LFS Batch API
 
+Added: v1.0
```

Comment:
Was this released in v1.0? The first mention of the batch API I have seen in our release notes comes from [v0.6.0][1].

[1]: https://github.com/git-lfs/git-lfs/releases/tag/v0.6.0